src/com/goide/completion/GoAutoImportCompletionContributor.java

```diff
@@ -306,18 +325,19 @@ public boolean process(GoNamedElement element) {
       return true;
     }
 
+    private boolean samePackage(@NotNull GoFile file, @Nullable String importPath) {
+      //noinspection ConstantConditions
```

Comment:
you rely on some complicated knowledges that are unreachable to IDE and not easy for understanding for developer. I bet it will burst one day.